docs/tutorials/platform_beginner/7_customize_login_and_register_forms.md

```diff
@@ -0,0 +1,212 @@
+# Step 7 - Customizing the user forms
+
+We want to have a consistent design in our web site. When using the pagelayout for the register forms, you see the same design as the rest of the website. We want to welcome our new users with a pretty register page.
+
+
+We want to have a consistent design over our web site. When using the register form, it was a raw HTML form without any styling on it. We want to welcome our new users with a pretty register page.
```

Comment:
This paragraph mostly repeats the previous one.

Reply:
https://media.giphy.com/media/EX6wKXVM9IOlO/giphy.gif